dashboard/config/scripts/levels/NetSim_SendBits.level

```diff
@@ -0,0 +1,12 @@
+<NetSim>
+  <config><![CDATA[{
+  "game_id": 42,
+  "created_at": "2015-01-20T02:25:10.000Z",
+  "level_num": "variant2",
+  "user_id": 1,
+  "properties": {
+    "showEncodingControls": ["none"],
```

Comment:
`"none"` is actually not a valid encoding; here, I had to put _something_ in the array, or `"showEncodingControls"` is not passed through to NetSim.  I have no idea why, but debugging from netsimMain it's clearly missing when the array is empty.  Do we have a better approach to this?